### ISSUE TITLE
AWS S3 bucket tags test

### ISSUE THREAD

**Issue by asilvanovich** [gruntwork-io/terratest#996]
Hi guys! Just started using terratest. I wanted to test that my AWS S3 bucket has certain keys in tags, but didn't find a function for this. There is a function [FindS3BucketWithTagE](https://github.com/gruntwork-io/terratest/blob/6d96ad5172920b1c81700987309b2100b8fa3821/modules/aws/s3.go#L25) that helps to find s3 bucket with the given key-value in tags, but I want something similar to [GetTagsForEc2InstanceE](https://github.com/gruntwork-io/terratest/blob/v0.37.10/modules/aws/ec2.go#L231), but for S3.

I can create the function, would you be interested in a PR?

**Comment by yorinasub17** [comment]
Yup a PR to add this would be appreciated! (for the same reason listed here https://github.com/gruntwork-io/terratest/issues/995#issuecomment-926946918)
